src/router/index.ts

```diff
@@ -1,10 +1,16 @@
 import { createRouter, createWebHistory } from "vue-router";
 
+const requiredAuthKeys = ["userId", "apiKey"];
+
 const routes = [
   {
     path: "/quiz/:quizId",
     name: "Player",
-    props: true,
+    props: (route: any) => ({
+      quizId: route.params.quizId,
+      thirdPartyUserId: route.query.userId,
+      thirdPartyApiKey: route.query.apiKey,
```

Comment:
why not just `userId` and `apiKey`?
All our authentication in quiz will be third party only.

This naming made sense in plio because it had it's own authentication system and a third party one as well